app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -103,148 +85,47 @@ We're available Monday to Friday between 8:30am and 5:30pm (UK local time) excep
 
-If you are not training to teach in these subjects, any support you get will depend on your immigration status. [Learn more about fees and funding for non-UK trainees](/non-uk-teachers/fees-and-funding-for-non-uk-trainees).
-
-## Find and apply for teacher training courses
-
-Your teacher training course must lead to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). Some courses combine QTS with a [postgraduate certificate in education (PGCE) or postgraduate diploma in education (PGDE)](/train-to-be-a-teacher/what-is-a-pgce). These are optional: you do not need a PGCE or PGDE to teach in England.
-
-You can train in a school or in a university setting, and study full-time (over a year) or part-time (over 2 or more years). Teacher training courses can also be 'fee-paying' or 'salaried'.
-
-### Fee-paying and salaried teacher training courses
-
-A fee-paying teacher training course means you pay a fee to study as a teacher. Most teacher trainees in England are enrolled on this type of course.
-
-Fee-paying courses include:
-
-* university teacher training programmes
-* school-centred initial teacher training (SCITT) programmes
-
-A salaried teacher training course means you are paid to work in a school while you learn to teach. However, these courses are limited in number and very competitive. Before you apply, contact the salaried teacher training provider to check you meet the entry requirements.
-
-Salaried courses include:
-
-* School Direct salaried teacher training programmes
-* the Teach First Leadership Development programme
-* postgraduate teaching apprenticeships
-
-[Find out more about salaried teacher training](/funding-and-support/salaried-teacher-training).
- 
-### Choosing a subject to teach 
-
-The subject you apply to teach must usually be the same as, or closely related to, the subject you studied at university. (For example, engineering graduates can apply to train to teach physics, but computer science graduates are unlikely to be accepted to train to teach geography.) 
-
-### Finding the right course for you and getting help
+[Learn more about fees and funding for non-UK trainees](/non-uk-teachers/fees-and-funding-for-non-uk-trainees).
 
-[Find postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) lists courses leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). 
+## Step 3: find and apply for teacher training courses
 
-You can also search courses by:
+If you’re applying for teacher training as a non-UK citizen, the process may be different to that for UK citizens.
 
-* location
-* teaching subject
-* age group
-* courses which pay a salary
-* courses for which student visa or skilled worker visa sponsorship is available
+Learn more and [read our tips on making an application to increase your chances of success](/tips-on-applying-for-teacher-training-as-a-non-uk-citizen).
```

Comment:
for some reason the path seems to be case sensitive - change it to: 
`/non-uk-teachers/tips-on-applying-for-teacher-training-as-a-non-UK-citizen`